fetch/api/basic/stream-safe-creation.any.js

```diff
@@ -0,0 +1,33 @@
+// These tests verify that stream creation is not affected by changes to
+// Object.prototype.
+
+const creationCases = {
+  fetch: async () => fetch(location.href),
+  request: () => new Request(location.href),
```

Comment:
I think I'm missing where [Request(url)](https://fetch.spec.whatwg.org/#dom-request) constructs a ReadableStream.

Reply:
IIRC, it actually does in Blink. But I changed the test case to one which always constructs a ReadableStream by the standard.